addons/supplies/functions/fnc_medicalArsenal.sqf

```diff
@@ -0,0 +1,75 @@
+#include "script_component.hpp"
+/*
+ * Author: JoramD
+ * Add medical arsenal to object.
+ *
+ * Arguments:
+ * 0: Object <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [object] call tac_supplies_fnc_medicalArsenal
+ *
+ * Public: No
+ */
+
+params ["_object"];
+
+GVAR(trainings) = false;
+
+private _medicalItems = [
+    "ACE_packingBandage",
+    "ACE_elasticBandage",
+    "ACE_fieldDressing",
+    "ACE_bloodIV",
+    "ACE_bloodIV_250",
+    "ACE_bloodIV_500",
+    "ACE_bodyBag",
+    "ACE_epinephrine",
+    "ACE_morphine",
+    "tac_medical_painkillers",
+    "ACE_plasmaIV",
+    "ACE_plasmaIV_250",
+    "ACE_plasmaIV_500",
+    "ACE_salineIV",
+    "ACE_salineIV_250",
+    "ACE_salineIV_500",
+    "ACE_splint",
+    "ACE_surgicalKit",
+    "ACE_tourniquet",
+    "ACE_adenosine",
+    "ACE_quikclot"
+];
+
+[_object, _medicalItems, true] call ACEFUNC(arsenal,addVirtualItems);
+
+private _action = [
+    QGVAR(medicalArsenal),
+    "Medical Arsenal",
+    "",
+    {
+        params ["_target", "_player"];
+
+        GVAR(trainings) = false;
+        [_target, _player, false] call ACEFUNC(arsenal,openBox);
+    },
+    {
+        params ["_target", "_player"];
```

Comment:
We should put this entire condition code into Mods `apollo` as an API.

Didn't think it through yet, but for example:
- Mods has function `getTraining`, returns if training is set or not
  - Could handle caching up to X seconds to not lag anything with repeated actions
- This should just call `["medic"] call tac_apollo_fnc_getTraining` and work off of that alone

Reply:
Entire function kept in Mods - https://github.com/Theseus-Aegis/Mods/pull/480